### DISCUSSION TITLE
Controller node

### DISCUSSION THREAD

**Discussion by senthil13** [provectus/kafka-ui#3635] [Q&A] [answered]
Does Kafka-ui show which node is a controller node? I don't see any documents on this topic so checking with experts.

**Comment by Haarolean** [comment, marked as answer]
Hi, yes, it's marked with a green check
<img width="664" alt="image" src="https://user-images.githubusercontent.com/1494347/230654708-2f244aa4-09b1-45ca-8458-d570f15eb6c5.png">
